### ISSUE TITLE
Ditto 3.24.238.0 hot key not working Windows 11 

### ISSUE THREAD

**Issue by rmhelvey** [sabrogden/Ditto#377]
I have been using Ditto for several years, recommending it to many colleagues. I moved to Windows 11 about 4 months ago and Ditto continued working well.  About one week ago the paste hot key Alt-V (the only feature I use) stopped working. I removed and reloaded Ditto and continue having the problem. Any suggestions?

OS Name	Microsoft Windows 11 Home
Version	10.0.22621 Build 22621
System Type	x64-based PC
System SKU	LENOVO_MT_81Q6_BU_idea_FM_Legion Y545
Processor	Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz, 2592 Mhz, 6 Core(s), 12 Logical Processor(s)
BIOS Version/Date	LENOVO BHCN28WW, 5/28/2019

Ditto Version 3.24.238.0

Attachment - shows Alt-V screen with ditto running 

![2022-11-28 09_05_19-Untitled - Notepad](https://user-images.githubusercontent.com/118945370/204326991-80d86ac0-4c50-4ba8-96a5-2ba259875c62.png)

**Comment by Chealer** [comment]
Does this persist with 03.25.113? ~~If so, what is that hotkey?~~

**Comment by rmhelvey** [comment]
I determine it is poor practice to use Alt as a hot key. Experimenting with
other key combinations I find I don't see the menu of recent clipboard
items, how do I get to that?

Thanks

Rick Helvey

**Comment by Chealer** [comment]
Sorry for my last question, you had already described the hotkey.

> Experimenting with other key combinations I find I don't see the menu of recent clipboard items, how do I get to that?

Do you mean the Quick Paste menu, which you can bring up by clicking Ditto's systray icon??

Is your issue specific to Notepad? And again, does this persist with version 03.25.113 or ulterior?

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>

**Comment by rmhelvey** [comment]
Yes, I am on 03.25.113.0. I have been using Alt V for quick paste for years
(and recommending Ditto as well) I wasn't paying  attention before but kept
getting frustrated when doing Alt V and having that problem i reached out
to you when i realized it was Alt calling the menu. So I want the quick
paste functionality that brings up this menu without using Alt V.  I looked
at the Quick Paste options, there are many of them but i didn't see simply
pulling the screen showed below.  Thanks

[image: image.png]

**Comment by Chealer** [comment]
GitHub is already flaky enough when you use its canonical interface; please do not reply via email. Your image does not show.

Sorry but your response is unclear. Is your issue specific to Notepad?
If you are saying this persists in 03.25.113, please retitle the ticket accordingly.
